sematic/db/queries.py

```diff
@@ -230,7 +232,6 @@ def save_run(run: Run) -> Run:
     Run
         saved run
     """
-    _assert_external_jobs_not_removed([run])
```

Comment:
Why this change?

Reply:
The run objects no longer have external jobs inside them, so there's no need to check on run-save that no external jobs have been removed.